fast64_internal/oot/oot_level_writer.py

```diff
@@ -84,6 +84,12 @@ def ootExportSceneToC(originalSceneObj, transformMatrix, f3dType, isHWv1, sceneN
         if exportSubdir == "":
             raise PluginError("Scene folder " + sceneName + " cannot be found in the ootSceneDirs list.")
 
+    roomObjects = [
+        obj for obj in bpy.data.objects if ((obj.parent == originalSceneObj) and (obj.ootEmptyType == "Room"))
+    ]
+    for roomObj, room in zip(roomObjects, scene.rooms.values()):
+        ootData.objectData.addMissingObjectsToList(roomObj, room, ootData.actorData, 0, None)
+        ootData.objectData.addAltHeadersObjects(roomObj, room, ootData.actorData)
```

Comment:
I also feel like this `addMissingObjectsToList` could go into `addAltHeadersObjects`
and `addAltHeadersObjects` -> `addRoomHeadersObjects`